deprecated.go

```diff
@@ -0,0 +1,105 @@
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	DirIterator = files.DirIterator
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	Directory = files.Directory
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	FileInfo = files.FileInfo
+)
+
+// Structs
+type (
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	Filter = files.Filter
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	Symlink = files.Symlink
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	MultiFileReader = files.MultiFileReader
+	ReaderFile      = files.ReaderFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	SliceFile = files.SliceFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	TarWriter = files.TarWriter
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	WebFile = files.WebFile
+)
+
+// Helpers
+var (
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	WriteTo = files.WriteTo
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewFilter = files.NewFilter
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewLinkFile = files.NewLinkFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	ToSymlink = files.ToSymlink
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewMultiFileReader = files.NewMultiFileReader
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewFileFromPartReader = files.NewFileFromPartReader
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewBytesFile = files.NewBytesFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewReaderFile = files.NewReaderFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewReaderStatFile = files.NewReaderStatFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewReaderPathFile = files.NewReaderPathFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewSerialFile = files.NewSerialFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewSerialFileWithFilter = files.NewSerialFileWithFilter
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	FileEntry = files.FileEntry
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewMapDirectory = files.NewMapDirectory
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewSliceDirectory = files.NewSliceDirectory
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewTarWriter = files.NewTarWriter
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	ToFile = files.ToFile
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	ToDir = files.ToDir
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	FileFromEntry = files.FileFromEntry
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	DirFromEntry = files.DirFromEntry
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	Walk = files.Walk
+	// Deprecated: moved to github.com/ipfs/go-libipfs/files
+	NewWebFile = files.NewWebFile
```

Comment:
I usually like to keep the old file structure (\*only files that had public symbols).

This makes the review process much easier.

Reply:
I separated by commits. You can go back earlier and see. I'm also happy to revert if that's better